bitnami/etcd/templates/prometheusrule.yaml

```
@@ -0,0 +1,26 @@
+{{- if and .Values.metrics.enabled .Values.metrics.prometheusRule.enabled  }}
```

Minor coding style improvement:

```suggestion
{{- if and .Values.metrics.enabled .Values.metrics.prometheusRule.enabled }}
```